pkg/util/webhooks/validating-webhooks/validating-webhook.go

```diff
@@ -15,7 +15,7 @@ import (
 	"kubevirt.io/client-go/log"
 )
 
-type admitter interface {
+type Admitter interface {
```

Comment:
Interfaces should rarely be exported. They exist to decouple packages dependencies, so it is less common to use them in other places like this.

It is fine to re-define the interface in the place it is needed (e.g. unit tests).
An alternative, which is pretty rare as well, is to define a package that only defines interfaces (there are such examples in the STD library).